java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/AbstractThreadedObservablePlugin.java

```diff
@@ -25,10 +27,19 @@
  * The {@link AbstractThreadedObservablePlugin} class provides the means to observe a plugin
  * (insertion/removal of readers) using a monitoring thread.
  */
-public abstract class AbstractThreadedObservablePlugin extends AbstractPlugin {
+public abstract class AbstractThreadedObservablePlugin extends AbstractPlugin
+        implements ObservablePlugin {
     private static final Logger logger =
             LoggerFactory.getLogger(AbstractThreadedObservablePlugin.class);
 
+    /* The observers of this object */
+    private Set<ObservablePlugin.PluginObserver> observers;
```

Comment:
The observers can't be managed though a 'Set', but only though a 'List'.
Because the observers are unknown objects. A Set collection requires that the objects implements the methods 'equals': impossible if the objects are unknown.